### PR TITLE
Feat: Added modal and rating

### PR DESCRIPTION
**Pull request by GauravDe-git**
image
image

### CODE REVIEW

**Comment by aman11s** [comment]
Screenshot 2022-02-21 at 8 50 20 PM

Modal is not opening.

**Comment by GauravDe-git** [comment]
> Screenshot 2022-02-21 at 8 50 20 PM
> 
> Modal is not opening.

Not able to figure out the issue , my brain is exhausted, oufff

File: Pages/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.checked{
+    color: orange;
+}
```
Comment (GeetaPogul) [inline Pages/Rating/rating.css:3]: use color variables

**Review by GeetaPogul — APPROVED** [review]
looks good to me go to be merge

**Review by anuj-kumary — APPROVED** [review]

**Pull request merged**